frontend/src/metabase/dashboard/components/DashCard/DashCardVisualization.tsx

```diff
@@ -534,6 +534,7 @@ export function DashCardVisualization({
             question={question}
             result={result}
             dashcard={dashcard}
+            canEdit={!isVisualizerDashboardCard(dashcard)}
```

Comment:
would it be easy to find out if there's only a single backing card? maybe we could disable only for dashcards with multiple cards visualized?

Reply:
That would be nice but due to the way visualizer cards work it would still be confusing. Even without adding additional cards in the visualizer by just editing a viz setting or display type it becomes "detached" from the original card so that editing the original card viz settings does not propagate them